src/main/java/edu/kis/powp/jobs2d/drivers/adapter/Job2dDriverAdapter.java

```diff
@@ -5,14 +5,17 @@
 import edu.kis.legacy.drawer.shape.LineFactory;
 import edu.kis.powp.jobs2d.Job2dDriver;
 
+
 /**
  * driver adapter to drawer with several bugs.
  */
-public class MyAdapter extends DrawPanelController implements Job2dDriver {
+public class Job2dDriverAdapter extends DrawPanelController implements Job2dDriver {
```

Comment:
The name is better but it does not indicate the purpose of the adapter or what was actually adapted.
Inheritance is not used (no super drawLine calls). Should be removed!